.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -1,85 +1,53 @@
+# yamllint disable-line rule:document-start
 name: "Bug report"
-description: Report a bug found while using the Berty apps or packages.
+description: Report a bug found while using weshnet.
 labels: ["bug", "🔍 Ready for Review"]
 assignees:
-- peletron
+  - jefft0
 body:
   - type: checkboxes
     attributes:
       label: Is there an existing issue for this?
-      description: Please search to see if an issue already exists for the bug you encountered.
+      description: >-
+        Please search to see if an issue already exists for the bug you
+        encountered.
       options:
-      - label: I have searched the existing issues
-        required: true
-  - type: dropdown
-    id: product
-    attributes:
-      label: Berty product
-      description: What Berty product are you seeing the problem on?
-      options:
-        - Mobile app
-        - Desktop app
-        - CLI tools (berty mini, daemon, etc.)
-        - Code package
-    validations:
-      required: true
+        - label: I have searched the existing issues
+          required: true
   - type: input
-    id: product-version
+    id: package-version
     attributes:
-      label:  Berty product version
-      description: |+
-                  What is the version of this Berty product?
-
-                  Tip: You can find the version of the mobile and desktop apps in the settings.
-                  To display the CLI tool version, run `berty version`.
-      placeholder: e.g. 2.396.1
-    validations:
-      required: true
-  - type: dropdown
-    id: os
-    attributes:
-      label: OS
-      description: What OS are you seeing the problem on?
-      options:
-        - iOS
-        - Android
-        - Linux
-        - macOS
-        - Windows
-        - Other
```

Comment:
I think you should keep the OS version because we have in the codebase some build constraints depending on the OS version you are building. So it's useful to have this king of information.

Reply:
OK. I force pushed the change.